<launch>
	<arg name="uav_name" default="drone_1" />
	<arg name="ugv_name" default="drone_1" />
    
    
    <node name="offb_node" pkg="control" type="offb_node" output="screen"/>

    <include file="$(find control)/launch/generate_waypoints_node.launch">
        <arg name="uav_name" value="$(arg uav_name)" />
        <arg name="ugv_name" value="$(arg ugv_name)" />
	</include>

    <include file="$(find control)/launch/extract_lane_node.launch">
        <arg name="uav_name" value="$(arg uav_name)" />
	</include>
</launch>
